src/Shortcodes/ShortcodesParser.cs

```diff
@@ -128,17 +128,17 @@ private Shortcode ParseShortcode()
                 {
                     arguments ??= CreateArgumentsDictionary();
 
-                    arguments[argumentIndex.ToString()] = Character.DecodeString(new TextSpan(_scanner.Buffer, _result.Start + 1, _result.Length - 2)).ToString();
```

Comment:
This is probably wrong. Start is the beginning of the result, and Offset is the current position. So Offset should be stored as a local Start before the call to Read